scripts/pkg/debian/postinstall.sh

```diff
@@ -9,12 +9,26 @@ then
   test -n "$(initctl status cnspec | grep -o running)" && \
     echo " -> Restart cnspec service (init)" && \
     initctl restart cnspec
+
+  # if mondoo service is running, stop it and start cnspec
+  test -n "$(initctl status mondoo | grep -o running)" && \
+    echo " -> Replace mondoo service with cnspec service (init)" && \
+    initctl stop mondoo && \
```

Comment:
Do we not need an equivalent to `systemctl disable mondoo` for non-systemd init so that the service doesn't start up on boot any more?

Reply:
Is there a way to do that?

Reply:
Yes, I went ahead and added two sections into the postinstall script
1) if mondoo.service is running; stop it and start cnspec.service
2) if mondoo.service is enabled for boot; disable it and enable cnspec.service

So the end of the package upgrade should leave you with cnspec running if mondoo.service was running and cnspec.service enabled if mondoo.service was previously enabled.

Reply:
I understand from this conversation that the service should also be enabled for Upstart. But from the comments above and in the commit message, I read this is no longer true. I assume comments and commit message are correct?

How would we inform the user, that the service changed?
When mondoo was active as a service, wouldn't this result in a failure after the next reboot? Mondoo is trying to start as a service, but the mondoo binary isn't present any longer.

Reply:
We chatted about Upstart a bit yesterday (Tim and me), and we didn't want to spend too much time on Upstart since there really shouldn't be any users of it left at this point (unless you know of a supported distro that still uses it).